src/components/StyledModal.js

```diff
@@ -0,0 +1,72 @@
+import React, { Component } from 'react'
+import {
+  Modal,
+  ScrollView,
+  TouchableOpacity,
+  View
+} from 'react-native'
+import EStyleSheet from 'react-native-extended-stylesheet'
+import Icon from 'react-native-vector-icons/FontAwesome'
+
+class StyledModal extends Component {
+  render() {
+    return (
+      <View>
+        <Modal
+          animationType={'fade'}
+          transparent={true}
+          onRequestClose={() => {}}
```

Comment:
Yes, on android, according to the documentation:
http://facebook.github.io/react-native/releases/0.37/docs/modal.html